plugins/modules/template_workflow_manager.py

```diff
@@ -423,10 +423,27 @@
             suboptions:
               do_version:
                 description: DoVersion query parameter. If this flag is true, creates a new
```

Comment:
Can we remove "DoVersion query parameter"? Can you have a look on the below version?

`  description: Determines whether to create a new version of the project with the imported contents. If set to true and the project already exists, a new version will be created. If false, the operation will fail with a 'Project already exists' error if the project already exists.`